src/client/pkg/config/config.go

```diff
@@ -20,7 +29,8 @@ func Read() (*Config, error) {
 	var c *Config
 
 	// Read json file
-	if raw, err := ioutil.ReadFile(configPath); err == nil {
+	p := configPath()
+	if raw, err := ioutil.ReadFile(p); err == nil {
```

Comment:
Also I just noticed looking at our `cmdutil` package that there's a way to read config files using it too. Not sure if that'll make things easier but it might.